src/k8s/pkg/k8sd/api/cluster_remove.go

```diff
@@ -14,7 +14,7 @@ import (
 	"github.com/canonical/microcluster/state"
 )
 
-func postClusterRemove(m *microcluster.MicroCluster, s *state.State, r *http.Request) response.Response {
+func postClusterRemoveNode(m *microcluster.MicroCluster, s *state.State, r *http.Request) response.Response {
```

Comment:
the name of those functions should match the API endpoint.
```
POST /cluster/remove
```
--> `postClusterRemove`